atlasdb-cassandra-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueServiceSweeperTest.java

```diff
@@ -28,7 +28,7 @@ protected KeyValueService getKeyValueService() {
         return CQLKeyValueService.create(
                 CassandraKeyValueServiceConfigManager.createSimpleManager(
```

Comment:
Yes, I think it makes sense to factor out external configuration for both Cassandra and CQL kvs. I will probably refactor the tests once docker is up and running.